### DISCUSSION TITLE
How to center align items in screen in newer version

### DISCUSSION THREAD

**Discussion by joeDH00** [Budibase/budibase#13934] [Help] [answered]
I am using budibase version 2.8.2 and there is this setting for the screen

![image](https://github.com/Budibase/budibase/assets/159232925/7507ccad-7aa4-41e1-8c84-22924f671ff1)

I am trying out version 2.13.14 and cant find the setting anywhere?

**Comment by ConorWebb96** [comment]
Hey @joeDH00,

I do believe those settings do still exist. It might be that the screen was moved to inside the component tree and you have to access it that way instead?

This is a very old version of Budibase. So apologies if I can't be more specific.

**Reply by joeDH00** [reply, marked as answer]
The settings could not be found due to my app running older version when the budibase itself is running a newer version. Upgrading to match the version did solve it. I decided to stay in version 2.8.2 as of now since there are few compatibility issues
